mkdocs/purpur/configuration.md

```diff
@@ -889,7 +889,7 @@ For a more clear explanation of the world settings section of the config, feel f
 * ##### deactivate-by-redstone
     - **default**: false
     - **description**: Allow spawners to be deactivated by redstone
-* ##### fix-mc-238526
+* ##### fix-mс-238526
```

Comment:
```suggestion
* ##### fix-mc-238526
```

Not a good solution. Since it's a header (which automatically creates a fragment) it needs to be something that can easily be typed without the use of special characters.